src/SearchModal.tsx

```diff
@@ -41,7 +41,7 @@ export const SearchModal = forwardRef(
       <VuiPortal>
         <div className="vrsStyleWrapper">
           {isOpen && (
-            <div style={{ zIndex }}>
+            <div style={{ zIndex, position: "fixed" }}>
```

Comment:
Nit: Can we combine this with the parent `vrsStyleWrapper` div? That would enable us to specify `position: "fixed"` on the class, which will be a little more maintainable because we typically use classes to define styles, reserving `style` prop for dynamic styles.

```jsx
          {isOpen && (
            <div className="vrsStyleWrapper" style={{ zIndex }}>
```